device_backends/DummyBackend/src/DummyBackend.cc

```diff
@@ -102,10 +119,25 @@ namespace ChimeraTK {
     return (_readOnlyAddresses.find({bar, address}) != _readOnlyAddresses.end());
   }
 
+  void DummyBackend::setReadCallbackFunction(
+      AddressRange addressRange, boost::function<void(void)> const& readCallbackFunction) {
+    _readCallbackFunctions.insert(
+        std::pair<AddressRange, boost::function<void(void)>>(addressRange, readCallbackFunction));
+  }
+
+  void DummyBackend::runReadCallbackFunctionsForAddressRange(AddressRange addressRange) {
+    std::list<boost::function<void(void)>> callbackFunctionsForThisRange =
+        findReadCallbackFunctionsForAddressRange(addressRange);
+    for(std::list<boost::function<void(void)>>::iterator functionIter = callbackFunctionsForThisRange.begin();
```

Comment:
Improve readability / modernise C++:
```
for(auto &function : callbackFunctionsForThisRange) {
  function();
}
```

Reply:
OK. I changed 
`void DummyBackend::runReadCallbackFunctionsForAddressRange`
and 
`void DummyBackend::runWriteCallbackFunctionsForAddressRange`